docs/fsdpv2.md

```diff
@@ -31,8 +31,30 @@ loss = output.sum()
 loss.backward()
 optim.step()
 ```
-It is also possible to shard individual layers separately and have an outer wrapper handle any leftover parameters. The autowrapping
-feature will come in the future releases.
+It is also possible to shard individual layers separately and have an outer wrapper handle any leftover parameters. Here is an example to autowrao each `DecoderLayer`.
+```python3
+from torch_xla.distributed.fsdp.wrap import transformer_auto_wrap_policy
+
+# Apply FSDP sharding on each DecoderLayer layer.
+auto_wrap_policy = functools.partial(
+    transformer_auto_wrap_policy,
+    transformer_layer_cls={
+        decoder_only_model.DecoderLayer
+    },
+)
+model = FSDPv2(
+    model, mesh=mesh, auto_wrap_policy=auto_wrap_policy)
+```
+You can also use the `size_based_auto_wrap_policy` which will auto wrap each layer if the parameters in its subtree exceed certain amount.
```

Comment:
Let's omit this as I haven't found it to be useful at all...

Reply:
sure